ios/fastlane/lanes/lane_build.rb

```diff
@@ -20,7 +20,7 @@
   clean_build_artifacts
 
   increment_build_number(
-    build_number: ENV["BUILD_NUMBER"], # based on commit, defined in GA
+    build_number: ENV.fetch("BUILD_NUMBER", nil), # based on commit, defined in GA
```

Comment:
Thank you for the contribution!
Although I would prefer to use `ENV.fetch` without a default value, otherwise, it's identical to what we carrently have.